### DISCUSSION TITLE
Problems packaging my jspsych experiment into psiturk

### DISCUSSION THREAD

**Discussion by radrogue1** [jspsych/jsPsych#1037] [Q&A] [answered]
Hi all,

I currently am attempting to run my jspsych experiment in psiturk in order to actually run the experiment. I have it fully packaged into the psiturk format, but when I run my experiment, I get the error that I have screenshotted and attached. My theory is that the timeline variables I am using are causing issues in the psiturk format. I think this because I am getting an error with the html-keyboard-response plugin when I use timeline variables, but when I do not the error is not thrown.

Thanks in advance for the help,
Ricardo
<img width="625" alt="Screen Shot 2020-09-09 at 2 11 51 PM" src="https://user-images.githubusercontent.com/68128449/92638271-045f3f80-f2a8-11ea-9576-33f0f76f3145.png">

**Comment by jodeleeuw** [comment, marked as answer]
Hey Ricardo,

It looks like you are adding objects to the main timeline with properties like `DFRtimeline` and `CVLTtimeline`. Those names need to be just `timeline`, which acts as a special keyword for jsPsych.

**Comment by radrogue1** [comment]
That worked! Thank you for your help.
